sqldelight-gradle-plugin/src/main/kotlin/app/cash/sqldelight/gradle/SqlDelightDatabase.kt

```diff
@@ -128,6 +128,10 @@ abstract class SqlDelightDatabase @Inject constructor(
     dependencies.add(database)
   }
 
+  fun srcDirs(vararg srcPaths: Any) {
+    srcDirs.setFrom(srcPaths)
```

Comment:
setFrom overwrites existing sources. The java and kotlin behavior (but I prefer java over kotlin) does not overwrite the existing sources but append them only. Just use `from` :)

Reply:
Changed. Now I know how `srcDirs` and `setSrcDirs` do under the hood, TIL